languageserver/src/main/scala/langserver/core/LanguageServer.scala

```diff
@@ -21,6 +21,9 @@ class LanguageServer(inStream: InputStream, outStream: OutputStream) extends Laz
         completionRequest(textDocument, position)
       case ("textDocument/definition", TextDocumentDefinitionRequest(TextDocumentPositionParams(textDocument, position))) =>
         gotoDefinitionRequest(textDocument, position)
+      case ("textDocument/hover", TextDocumentDefinitionRequest(TextDocumentPositionParams(textDocument, position))) =>
```

Comment:
yeah now I'm puzzled too, since it's not working anymore. I think was one of those fortuitous moments that it worked "on its own" that I mistook for actually making it working myself